src/Miunie.Core/Json/IJsonParser.cs

```diff
@@ -0,0 +1,22 @@
+﻿// This file is part of Miunie.
+//
+//  Miunie is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  Miunie is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with Miunie. If not, see <https://www.gnu.org/licenses/>.
+
+namespace Miunie.Core.Json
+{
+    public interface IJsonParser
+    {
+        string ConvertToJson<T>(T obj);
```

Comment:
And for this I'd say `Serialize<T>`. Basically the same as both Newtonsoft JSON and System.Text.Json.

It seems to be a standard so why not use it. :blush: